scanpy/datasets/__init__.py

```diff
@@ -295,3 +295,107 @@ def pbmc3k_processed() -> AnnData:
         backup_url='https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/example-dataset/pbmc3k.h5ad',
     )
     return adata
+
+def SGE10x(sample_ID = 'V1_Breast_Cancer_Block_A_Section_1') -> AnnData:
+    """Processed Spatial Gene Expression data from 10x Genomics.
+    Database: https://support.10xgenomics.com/spatial-gene-expression/datasets
+    Parameters
+    ----------
+    sample_ID
+        The ID of data sample in 10x spatial data base.
+        It should be one of the followings:       
+        'V1_Breast_Cancer_Block_A_Section_1'
+        'V1_Breast_Cancer_Block_A_Section_2'
+        'V1_Human_Heart'
+        'V1_Human_Lymph_Node'
+        'V1_Mouse_Kidney'
+        'V1_Adult_Mouse_Brain'
+        'V1_Mouse_Brain_Sagittal_Posterior'
+        'V1_Mouse_Brain_Sagittal_Posterior_Section_2'
+        'V1_Mouse_Brain_Sagittal_Anterior'
+        'V1_Mouse_Brain_Sagittal_Anterior_Section_2'
+        By defaults, It is set to:
+        'V1_Breast_Cancer_Block_A_Section_1'
+
+    Returns
+    -------
+    Annotated data matrix.
+    """    
+    
+    
+    # setting filenames, tarfilenames, backup_rurls
+    # tarfilenames and backup_urls will be used for downloading data base
+    from urllib.parse import urljoin
+    files = {
+        'counts'                 : settings.datasetdir / (sample_ID + '_raw_feature_bc_matrix.h5'),
```

Comment:
please use `f'{sample_id}_raw_feature_bc_matrix.h5'` here and in similar cases

Reply:
Done.